src/components/Atoms/ReusableInput.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components'
+import React from "react";
+import { UserLoginReq } from 'types';
```

Comment:
chyba nie ma takiego export jak UserLoginReq ?

Reply:
import jest z typowania z BE boi tak to działa, żeby go mieć musisz pobrać mojego branch 'M1Connect' albo sb przekleić z github